### PR TITLE
Optimizing the Security of your Skywallet.md

### PR DESCRIPTION
**Pull request by sreekumar13**

### CODE REVIEW

File: Optimizing the Security of your Skywallet.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Optimizing the Security of your Skywallet
+
+Even though the Skywallet provides greater security when compared to a Desktop Wallet, it is essential to follow these security optimizations to get maximum security benefit from your Skywallet.
+
+Two things are necessary for a third party to access your funds in Skywallet, which is either the PIN code of the Skywallet or the Seed of the Skywallet.
```
Comment (jdknives) [inline Optimizing the Security of your Skywallet.md]: `One of two things is necessary...` ; `which is either the PIN code  of the  Skywallet and access to the Skywallet...`

File: Optimizing the Security of your Skywallet.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+You are recommended to create a secure PIN code, although, unless your Skywallet is lost or got stolen, no one can access it using the PIN code.
```
Comment (jdknives) [inline Optimizing the Security of your Skywallet.md]: Just say, `it is recommended to create a secure...` .

File: Optimizing the Security of your Skywallet.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## PIN Code
+
+Your PIN code is a four-digit combination, and a brand new Skywallet which you receive via shipment does not have a PIN code.
+
+Hence, it is essential first to set a PIN code right after you receive your Skywallet. If you want to know more about how to set the PIN code and personalize your Skywallet, please visit the user manual [Personalization of your Skywallet](https://github.com/skycoin/hardware-wallet/wiki/Getting-to-know-the-wallet).
+
+Though the PIN code is a four-digit combination, it would not be easy for an attacker/hacker to try the different combinations and work it out. 
+
+Because after each failed attempt, one has to wait for a specific time before attempting the next combination and with each failed attempt the waiting time increases exponentially, making it impossible for an attacker/hacker to obtain the PIN code that way.
```
Comment (jdknives) [inline Optimizing the Security of your Skywallet.md]: , it is very unlikely that an attacker can gain access to  your device this way.

File: Optimizing the Security of your Skywallet.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+However, such a security precaution is ineffective if you create the pin as an easy to guess one.
```
Comment (jdknives) [inline Optimizing the Security of your Skywallet.md]: if your PIN  is easy to guess. Watch out for consistent capitalization.

File: Optimizing the Security of your Skywallet.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+Hence, it is essential first to set a PIN code right after you receive your Skywallet. If you want to know more about how to set the PIN code and personalize your Skywallet, please visit the user manual [Personalization of your Skywallet](https://github.com/skycoin/hardware-wallet/wiki/Getting-to-know-the-wallet).
+
+Though the PIN code is a four-digit combination, it would not be easy for an attacker/hacker to try the different combinations and work it out. 
+
+Because after each failed attempt, one has to wait for a specific time before attempting the next combination and with each failed attempt the waiting time increases exponentially, making it impossible for an attacker/hacker to obtain the PIN code that way.
+
+However, such a security precaution is ineffective if you create the pin as an easy to guess one. 
+
+Examples include but not limited to, important dates, or sequences (like 1234) or repeating digits(1111).
```
Comment (jdknives) [inline Optimizing the Security of your Skywallet.md]: but are not limited to

File: Optimizing the Security of your Skywallet.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+Hence a secure PIN code is something easy to remember and at the same time not easy to guess for a third party.
+
+## Seed
+
+A Seed of the Skywallet is even more critical than a PIN code, as a hacker/malware without access to your Skywallet physically, can remotely access or steal your fund just using the Seed.
+
+As the Seed is not created by you and is a generated one by the device, you need not worry about the strength of the Seed as such.
```
Comment (jdknives) [inline Optimizing the Security of your Skywallet.md]: is generated  on the device

File: Optimizing the Security of your Skywallet.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+**However, it is imperative to keep the backup of the Seed in a safe and accessible place, as it is the only way to access your funds if anything happens to your Skywallet.**
+
+Following are some recommendations to backup and store your Seed safely:
+
+* **Do not Store the Seed in an electronic device. Because storing a Seed in an electronic device effectively nullifies the security it ensures, as the electronic device is hackable by a potential hacker and can get access to your Seed.**
```
Comment (jdknives) [inline Optimizing the Security of your Skywallet.md]: can potentially be hacked by  an attacker, thereby giving access to your Seed.

File: Optimizing the Security of your Skywallet.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+Hence it is recommended to store the Seed in a paper or a notebook which can then be stored in a safe place that is inaccessible to any unauthorized person at the same time not prone to any accidents.<br />
+**The only instance when the Seed is entered into a computer is when you are restoring the backup of your Wallet**, in that case, the Skywallet and Desktop Wallet process the Seed in a secure way which ensures that a third party will not gain access to your Seed.<br />
+The process is explained in detail in the section (Restore Backup)[https://github.com/skycoin/hardware-wallet/wiki/Restore-configuration].
+
+* Create multiple copies of the Seed and store it in separate locations.
```
Comment (jdknives) [inline Optimizing the Security of your Skywallet.md:39]: Are these recommendations made by  Trezor?

**Review by jdknives — CHANGES_REQUESTED** [review]
Why are you using <br> here? Would a newline not be enough? HTML sometimes makes rendering on different websites  different. Markdown only would be more consistent.

File: Optimizing the Security of your Skywallet.md (hunk 8 above)
Comment (sreekumar13) [reply Optimizing the Security of your Skywallet.md:39]: The first 3 are actually recommended by Trezor, however I took the last point from the existing wiki. I felt that is also a valid option, however, if you feel that would confuse the user, I can scrap that last point.

Comment (jdknives) [reply Optimizing the Security of your Skywallet.md:39]: No. Should be fine. Thanks for explaining.

File: Optimizing the Security of your Skywallet.md [hunk 9]
```diff
@@ -2,9 +2,9 @@
 
 Even though the Skywallet provides greater security when compared to a Desktop Wallet, it is essential to follow these security optimizations to get maximum security benefit from your Skywallet.
 
-Two things are necessary for a third party to access your funds in Skywallet, which is either the PIN code of the Skywallet or the Seed of the Skywallet.
+One of two things is necessary for a third party to access the Skywallet, which is either the PIN code of the Skywallet and the Seed of the Skywallet.
```
Comment (jdknives) [inline Optimizing the Security of your Skywallet.md]: Still not accurate. They either need Skywallet + PIN **OR** Seed.

File: Optimizing the Security of your Skywallet.md [hunk 10]
```diff
@@ -2,9 +2,9 @@
...
 
-You are recommended to create a secure PIN code, although, unless your Skywallet is lost or got stolen, no one can access it using the PIN code.
+It is recommended to create a secure PIN code, although, unless your Skywallet is lost or got stolen, no one can access it using the PIN code.
```
Comment (jdknives) [inline Optimizing the Security of your Skywallet.md]: Reading it again, it is best to just say to use a secure PIN. The second part of the sentence might disincentivize people to setup a secure PIN.

File: Optimizing the Security of your Skywallet.md [hunk 11]
```diff
@@ -18,31 +18,31 @@ Hence, it is essential first to set a PIN code right after you receive your Skyw
 
 Though the PIN code is a four-digit combination, it would not be easy for an attacker/hacker to try the different combinations and work it out. 
 
-Because after each failed attempt, one has to wait for a specific time before attempting the next combination and with each failed attempt the waiting time increases exponentially, making it impossible for an attacker/hacker to obtain the PIN code that way.
+Because after each failed attempt, one has to wait for a specific time before attempting the next combination and with each failed attempt the waiting time increases exponentially thus making it highly unlikely for an attacker/hacker to obtain access of your Skywallet this way.
```
Comment (jdknives) [inline Optimizing the Security of your Skywallet.md]: obtain access to your Skywallet.

File: Optimizing the Security of your Skywallet.md [hunk 12]
```diff
@@ -18,31 +18,31 @@ Hence, it is essential first to set a PIN code right after you receive your Skyw
 
 Though the PIN code is a four-digit combination, it would not be easy for an attacker/hacker to try the different combinations and work it out.
```
Comment (jdknives) [inline Optimizing the Security of your Skywallet.md]: Although the PIN code is only a combination of four digits, it is very unlikely for an attacker to guess a secure PIN. 

Then continue with the next paragraph.

File: Optimizing the Security of your Skywallet.md [hunk 13]
```diff
@@ -18,31 +18,31 @@ Hence, it is essential first to set a PIN code right after you receive your Skyw
 
 Though the PIN code is a four-digit combination, it would not be easy for an attacker/hacker to try the different combinations and work it out. 
 
-Because after each failed attempt, one has to wait for a specific time before attempting the next combination and with each failed attempt the waiting time increases exponentially, making it impossible for an attacker/hacker to obtain the PIN code that way.
+Because after each failed attempt, one has to wait for a specific time before attempting the next combination and with each failed attempt the waiting time increases exponentially thus making it highly unlikely for an attacker/hacker to obtain access of your Skywallet this way.
 
-However, such a security precaution is ineffective if you create the pin as an easy to guess one. 
+However, such a security precaution is ineffective if you create the PIN as an easy to guess one.
```
Comment (jdknives) [inline Optimizing the Security of your Skywallet.md]: if you create an insecure PIN.

File: Optimizing the Security of your Skywallet.md [hunk 14]
```diff
@@ -18,31 +18,31 @@ Hence, it is essential first to set a PIN code right after you receive your Skyw
...
 Though the PIN code is a four-digit combination, it would not be easy for an attacker/hacker to try the different combinations and work it out. 
 
-Because after each failed attempt, one has to wait for a specific time before attempting the next combination and with each failed attempt the waiting time increases exponentially, making it impossible for an attacker/hacker to obtain the PIN code that way.
+Because after each failed attempt, one has to wait for a specific time before attempting the next combination and with each failed attempt the waiting time increases exponentially thus making it highly unlikely for an attacker/hacker to obtain access of your Skywallet this way.
 
-However, such a security precaution is ineffective if you create the pin as an easy to guess one. 
+However, such a security precaution is ineffective if you create the PIN as an easy to guess one. 
 
-Examples include but not limited to, important dates, or sequences (like 1234) or repeating digits(1111).
+Examples include but are not limited to, important dates, or sequences (like 1234) or repeating digits(1111).
```
Comment (jdknives) [inline Optimizing the Security of your Skywallet.md]: Examples of an insecure PIN include but are not limited to ...

**Review by jdknives — CHANGES_REQUESTED** [review]

**Pull request merged**